excel/Module.md

```diff
@@ -1,89 +1,23 @@
-    ```
-
-10. This will prompt you to enter the username and password for signing into the Azure Portal App.
-
-11. Once the username and password pair is successfully entered, this will give a URL as follows on the browser address bar.
-
-    `https://login.microsoftonline.com/common/oauth2/nativeclient?code=xxxxxxxxxxxxxxxxxxxxxxxxxxx`
-
-12. Copy the code parameter (`xxxxxxxxxxxxxxxxxxxxxxxxxxx` in the above example) and in a new terminal, enter the following cURL command by replacing the `<CODE>` with the code received from the above step. The `<CLIENT_ID>` and `<CLIENT_SECRET>` parameters are the same as above.
-
-    ```
-    curl -X POST --header "Content-Type: application/x-www-form-urlencoded" --header "Host:login.microsoftonline.com" -d "client_id=<CLIENT_ID>&client_secret=<CLIENT_SECRET>&grant_type=authorization_code&redirect_uri=https://login.microsoftonline.com/common/oauth2/nativeclient&code=<CODE>&scope=Files.ReadWrite offline_access" https://login.microsoftonline.com/common/oauth2/v2.0/token
-    ```
-
-    The above cURL command should result in a response as follows.
-    ```
-    {
-      "token_type": "Bearer",
-      "scope": "Files.ReadWrite",
-      "expires_in": 3600,
-      "ext_expires_in": 3600,
-      "access_token": "<ACCESS_TOKEN>",
-      "refresh_token": "<REFRESH_TOKEN>",
-    }
-    ```
-
-13. Provide the following configuration information in the `Config.toml` file to use the Microsoft Excel connector.
-
-    ```ballerina
-    clientId = <CLIENT_ID>
-    clientSecret = <CLIENT_SECRET>
-    refreshUrl = <REFRESH_URL>
-    refreshToken = <REFRESH_TOKEN>
-    workbookIdOrPath = <WORKBOOK_ID_OR_PATH>
-    ```
-
-    The `workbookIdOrPath` is workbook ID or file path (with the `.xlsx` extension from root. If you have a file in root directory with name of `Work.xlsx`, you need to pass `workbookIdOrPath` as `Work.xlsx`). Make sure you create a workbook in Microsoft OneDrive and pass the correct `workbookIdOrPath` before using the connector.
+## Prerequisites
+Before using this connector in your Ballerina application, complete the following:
+- Create [Microsoft Office365 account](https://signup.live.com/)
+- Obtain token - Follow [this link](https://docs.microsoft.com/en-us/graph/auth-v2-user#authentication-and-authorization-steps)
 
 ## Quickstart
+To use the Microsoft Excel connector in your Ballerina application, update the .bal file as follows:
 
-### Create a worksheet in Workbook
-#### Step 1: Import Excel module
+### Step 1: Import connector
 First, import the ballerinax/microsoft.excel module into the Ballerina project.
 ```ballerina
 import ballerinax/microsoft.excel;
 ```
-#### Step 2: Configure the connection credentials.
-You can now make the connection configuration using the OAuth2 refresh token grant config.
+### Step 2: Create a new connector instance
+Create a `excel:Configuration` with the OAuth2 tokens obtained, and initialize the connector with it. 
```

Comment:
Create an

Reply:
Remove the ,

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-microsoft.excel/pull/15/commits/3e16c3fcae95051a7c24ecef4a633a6e7f3e29ef